.changeset/twenty-peas-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/catalog-client': minor
```

Comment:
Let's make this a `patch`, we use that for non-breaking changes while we're on 0.x